packages/airnode-node/src/workers/local-handlers.ts

```diff
@@ -2,22 +2,25 @@ import * as path from 'path';
 import { ethers } from 'ethers';
 import { logger } from '@api3/airnode-utilities';
 import { go } from '@api3/promise-utils';
-import { loadTrustedConfig } from '../config';
+import { loadTrustedConfig, setEnvValue, Config } from '../config';
 import * as handlers from '../handlers';
 import * as state from '../providers/state';
 import { WorkerResponse, InitializeProviderPayload, CallApiPayload, ProcessTransactionsPayload } from '../types';
 
 function loadConfig() {
   return loadTrustedConfig(path.resolve(`${__dirname}/../../config/config.json`), process.env);
 }
+function setAirnodePrivateKeyToEnv(config: Config) {
```

Comment:
Nit: You can pass just the mnemonic, not the whole config.

Reply:
Fixed :+1: